modules/navier_stokes/test/tests/finite_element/ins/energy-conservation/tests

```diff
@@ -0,0 +1,19 @@
+[Tests]
+  issues = '#22074'
+  design = 'navier_stokes/index.md INSElementIntegralEnergyAdvection.md'
+  [conservation]
+    requirement = 'The system shall exhibit global conservation of energy when using the continuous-Galerkin finite element spatial discretization with streamline-upwind Petrov-Galerkin stabilization and with'
+    [q2q1]
+      type = Exodiff
```

Comment:
exodiff is overkill, csvdiff is more appropriate and will cost less memory

Reply:
jus trying to be consistent with what I m requesting from the reactor module PR makers

Reply:
:+1: